packages/upgrade/package.json

```diff
@@ -14,8 +14,12 @@
   "bugs": "https://github.com/carbon-design-system/carbon/issues",
   "files": [
     "bin",
-    "cli.js"
+    "cli.js",
+    "transforms"
   ],
+  "installConfig": {
+    "hoistingLimits": "workspaces"
+  },
```

Comment:
This was added to ensure that `jscodeshift` is available in the local workspace's `node_modules`. By default yarn was hoisting it up to the root and `npm-which` couldn't find it for some reason. This only impacts this workspace and shouldn't impact other workspaces/packages.